docs/Caliptra_rtl.md

```diff
@@ -0,0 +1,862 @@
+![OCP Logo](./images/OCP_logo.png)
+
+<p style="text-align: center;">Caliptra Integration Specification</p>
+
+<p style="text-align: center;">Version 0.8</p>
```

Comment:
When will this move to 1.0?

Reply:
@andreslagarcavilla thoughts on when to move to v1?